exist-core/src/main/java/org/exist/security/AbstractGroup.java

```diff
@@ -176,6 +172,22 @@ public void addManager(final String name) throws PermissionDeniedException {
         managers.add(new ReferenceImpl<>(getRealm().getSecurityManager(), "getAccount", name));
     }
 
+    //this method is used by Configurator
```

Comment:
Where exactly?

Reply:
Yeah it wasn't easy to find and fix! It is used by `Configurator#searchForInsertMethod(Class, String)`.